doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -19,16 +19,17 @@
 
 ## Synopsis
 
-Ensure Reference Implementation (RI) of CNTT Reference Model (RM) and CNTT Reference Architecture (RA) meets industry driven quality assurance standards for compliance, verification and validation.  The OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN), overseen by the Compliance Verification Committee (CVC), will provide tracking and governance for RM/RA verification.
+The goal of Reference Conformance (RC) is to ensure the Reference Implementation (RI) of CNTT Reference Architecture (RA) meets industry driven validation to ensure sufficient quality as an Industry Reference. Furthermore RC supports the conformance goals of the OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN) and overseen by the Compliance Verification Committee (CVC).
```

Comment:
I think we need to be clear whether it is the RC's responsibility to make sure RI meets industry quanlity, or it should be gating tests within OPNFV to make sure of this?

Reply:
I am still having concerns for the 'validation' words. I think we need to be clear whether RC is responsible for validation. According our last week's agreement, RC will only address requirement of 'must'. Then my understanding would be RC is only for conformance. We can put validation of RI in RI WS or in OPNFV projects. I think we probably need some discussion on our next RI & RC call.